typings/index.ts

```diff
@@ -398,13 +398,38 @@ client.on('ready', async () => {
   });
 });
 
+// This is to check that stuff is the right type
+declare const assertIsMember: (m: Promise<GuildMember>) => void;
```

Comment:
Perhaps name this `assertIsPromiseMember`? Although the one below (`assertIsMessage`) should probably be updated.

Reply:
Do we need to differentiate? I could union type `Promise<GuildMember> | GuildMember`

Reply:
We want type assertions to be as strict as possible, we probably don't want a method that should be returning a `Promise<T>` to accidentally return `T` 😄